src/main/java/com/SurveyMonkey/model/questions/Questions.java

```diff
@@ -6,7 +6,7 @@
 import java.util.HashMap;
 
 @Entity
-public class Questions {
+abstract public class Questions {
```

Comment:
There is already an abstract type called QType, this class was supposed to be what CreateSurvey is right now (a class to hold a list of questions).